src/pages/HomePage/pages/DashBoard/Components/EventCard.tsx

```diff
@@ -0,0 +1,246 @@
+import { useState } from 'react';
+import {
+  CalendarDaysIcon,
+  ClockIcon,
+  MapPinIcon,
+  CalendarIcon,
+  ChevronDownIcon
+} from '@heroicons/react/24/outline';
+
+// Event interface
+interface Event {
```

Comment:
naming convention 
we start interface names with "I" so it easy to sport in the code and types end with type

Reply:
also event_name is optional?????
and start_date